CanvasPlusPlayground/Features/Pinned Items/PinnedItem.swift

```diff
@@ -25,20 +27,20 @@ struct PinnedItem: Identifiable, Codable {
         }
     }
 
-    func itemData() async -> PinnedItemData? {
+    func itemData() async {
         do {
             async let modelData = try fetchData()
             async let course = try CanvasService.shared.loadAndSync(
                 CanvasRequest.getCourse(id: courseID)
             )
```

Comment:
use the onCacheClosure to make use of the cache:
```suggestion
            async let course = try CanvasService.shared.loadAndSync(
                CanvasRequest.getCourse(id: courseID),
                onCacheReceive: { /* ... */ } 
            )
```

Reply:
And the same needs to be done below in `fetchData()`

Reply:
Addressed in f704016580131461c490bf7937cb94c243ce22e5